src/components/ArchivePage/EventInfoItem.js

```diff
@@ -50,28 +53,54 @@ const useStyles = makeStyles(theme => ({
 	}
 }));
 
-function EventInfoItem({ name, mainLink, tags, directors, workshops, tagHighlight }) {
+function EventInfoItem({ name, mainLink, tags, directors, workshops, tagHighlight, slug }) {
 	const classes = useStyles();
-	const [isExpanded, setExpanded] = useState(true);
-	return <Accordion classes={{ root: classes.paperRoot }}
-		onChange={() => setExpanded(e => !e)}>
+	const [isExpanded, setExpanded] = useState(false);
+	console.log(name)
+	console.log(`window: `, window.location.hash)
+	const [hash, setHash] = useState(window.location.hash);
```

Comment:
I actually think generally we aren't able to use the `window` object in the function body except in a react useEffect. https://blog.greenroots.info/gatsby-the-window-is-not-defined-error-what-and-how-to-fix-it 

Could you try play around with it and see what works in yarn build?